slate.config.js

```diff
@@ -8,8 +8,13 @@ const alias = {
 };
 
 module.exports = {
-  extends: {
-    dev: {resolve: {alias}},
-    prod: {resolve: {alias}},
+  slateCssVarLoader: {
+    cssVarLoaderLiquidPath: [path.resolve(__dirname,'src/snippets/css-variables.liquid')],
```

Comment:
Do we need to include `path.resolve`?

Reply:
If I understood correctly, it generates the absolute path so this line will do : 

`/Users/myUsername/slate/src/snippets/css-variables.liquid`?

Reply:
Tested it without `path.resolve()` and without `__dirname`. Worked 👍. Will remove.

Reply:
Now that I think of it -- this should be in a seperate PR. Moving it.